src/cpp/src/utils.cpp

```diff
@@ -266,6 +266,42 @@ ov::Core singleton_core() {
     return core;
 }
 
+void print_compiled_model_properties(ov::CompiledModel& compiled_Model, ov::Core& core) {
```

Comment:
do we need to pass Core here?
We typically, create core internally, so all such Cores will print the same `EXECUTION_DEVICES`.

Reply:
updated